audeer/core/io.py

```diff
@@ -136,6 +136,16 @@ def create_archive(
         RuntimeError: if archive does not end with ``zip`` or ``tar.gz``
             or a file in ``files`` is not below ``root``
 
+    Examples:
+        >>> _ = touch('a.txt')
+        >>> _ = touch('b.txt')
+        >>> _ = create_archive('.', None, 'archive.zip')
+        >>> extract_archive('archive.zip', '.')
+        ['a.txt', 'b.txt']
+        >>> _ = create_archive('.', ['a.txt'], 'archive.tar.gz')
```

Comment:
`create_archive()` does not return a value:

```suggestion
        >>> create_archive('.', None, 'archive.zip')
        >>> extract_archive('archive.zip', '.')
        ['a.txt', 'b.txt']
        >>> create_archive('.', ['a.txt'], 'archive.tar.gz')
```